<ui version="4.0" >
 <class>WalkSiteWndClass</class>
 <widget class="QMainWindow" name="WalkSiteWndClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WalkSiteWnd</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Base URL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wsw_le_base_url" >
          <property name="text" >
           <string>http://localhost/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="wsw_pb_start_walk" >
          <property name="text" >
           <string>&amp;Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Save To:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="wsw_cb_save_to" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable" >
           <bool>true</bool>
          </property>
          <item>
           <property name="text" >
            <string>.</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="wsw_tb_show_save_path" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="wsw_setting_frame" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Max Size(MB):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="wsw_cb_time_stamp" >
             <property name="text" >
              <string>timestamping</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QSpinBox" name="wsw_sb_recursive_level" >
             <property name="minimum" >
              <number>-1</number>
             </property>
             <property name="value" >
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QSpinBox" name="wsw_sb_max_size" >
             <property name="maximum" >
              <number>1024</number>
             </property>
             <property name="minimum" >
              <number>0</number>
             </property>
             <property name="singleStep" >
              <number>10</number>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QCheckBox" name="wsw_cb_span_hosts" >
             <property name="text" >
              <string>span-hosts</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Recursive Level:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QCheckBox" name="wsw_cb_follow_ftp" >
             <property name="text" >
              <string>follow-ftp</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Thread Count:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QSpinBox" name="wsw_sb_thread_count" >
             <property name="maximum" >
              <number>50</number>
             </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="value" >
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="2" >
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QListView" name="wsw_lv_reject_host" >
               <property name="toolTip" >
                <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Rejected Host List&lt;/p>&lt;/body>&lt;/html></string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QPushButton" name="wsw_pb_add_reject_host" >
                 <property name="text" >
                  <string>&amp;Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wsw_pb_delete_reject_host" >
                 <property name="text" >
                  <string>&amp;Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0" >
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QListView" name="wsw_lv_accept_ext" >
               <property name="toolTip" >
                <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Accepted Extension List&lt;/p>&lt;/body>&lt;/html></string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QPushButton" name="wsw_pb_add_accept_ext" >
                 <property name="text" >
                  <string>&amp;Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wsw_pb_delete_accept_ext" >
                 <property name="text" >
                  <string>&amp;Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="2" >
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QListView" name="wsw_lv_accept_host" >
               <property name="toolTip" >
                <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Accepted Host List&lt;/p>&lt;/body>&lt;/html></string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QPushButton" name="wsw_pb_add_accept_host" >
                 <property name="text" >
                  <string>&amp;Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wsw_pb_delete_accept_host" >
                 <property name="text" >
                  <string>&amp;Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="1" colspan="2" >
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QListView" name="wsw_lv_reject_ext" >
               <property name="toolTip" >
                <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Rejected Extension List&lt;/p>&lt;/body>&lt;/html></string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" >
               <property name="margin" >
                <number>0</number>
               </property>
               <property name="spacing" >
                <number>6</number>
               </property>
               <item>
                <widget class="QPushButton" name="wsw_pb_add_reject_ext" >
                 <property name="text" >
                  <string>&amp;Add</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="wsw_pb_delete_reject_ext" >
                 <property name="text" >
                  <string>&amp;Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Complete Page:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wsw_le_retrive_page_num" >
          <property name="text" >
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Complete Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wsw_le_retrive_total_size" >
          <property name="text" >
           <string>0 B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Average Speed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wsw_le_retrive_speed" >
          <property name="text" >
           <string>0 KB/s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Error Page:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wsw_le_error_page_num" >
          <property name="text" >
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="graphicsView" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">这个地方画什么图呢？&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">画个雷达扫描图怎么样？感觉不错啊。&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView_2" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="wsw_te_walk_log" />
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManual" />
    <addaction name="separator" />
    <addaction name="actionAbout_NullGet" />
    <addaction name="actionAbout_WalkSite" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuHehe" >
    <property name="title" >
     <string>walk</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="separator" />
    <addaction name="actionStart" />
    <addaction name="actionStop" />
    <addaction name="actionRestart" />
    <addaction name="separator" />
    <addaction name="actionSave_As" />
    <addaction name="actionLoad" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuSite" >
    <property name="title" >
     <string>site</string>
    </property>
    <addaction name="actionExplorer_In_Site_Browser" />
    <addaction name="actionOpen_File" />
    <addaction name="actionOpen_Director" />
    <addaction name="separator" />
    <addaction name="actionSettings" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionShow_Radar_Scanner" />
   </widget>
   <addaction name="menuHehe" />
   <addaction name="menu_View" />
   <addaction name="menuSite" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionStart" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/start.PNG</iconset>
   </property>
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset>icons/pause.PNG</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionRestart" >
   <property name="text" >
    <string>Restart</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionLoad" >
   <property name="text" >
    <string>Load ...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset>icons/exit.PNG</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_WalkSite" >
   <property name="text" >
    <string>About WalkSite</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionManual" >
   <property name="text" >
    <string>Manual</string>
   </property>
  </action>
  <action name="actionExplorer_In_Site_Browser" >
   <property name="text" >
    <string>Explorer In Site Browser</string>
   </property>
  </action>
  <action name="actionOpen_File" >
   <property name="text" >
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Director" >
   <property name="text" >
    <string>Open Director</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset>icons/option.PNG</iconset>
   </property>
   <property name="text" >
    <string>Setting</string>
   </property>
  </action>
  <action name="actionAbout_NullGet" >
   <property name="text" >
    <string>About NullGet</string>
   </property>
  </action>
  <action name="actionNew" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionShow_Radar_Scanner" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Radar Scanner</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
